src/Core/Time/Chronometer.cpp

```diff
@@ -39,6 +44,6 @@ namespace obe::Time
 
     bool Chronometer::over() const
     {
-        return ((!m_started) xor (this->getTime() > m_limit));
+        return ((!started()) xor (this->getTime() > m_limit));
```

Comment:
```cpp
return m_started && this->getTime() > m_limit;
```
would be better :
- We check that the Chronometer has started (otherwise we return `false`)
- We check that the time is over the limit